main/src/ca/uwaterloo/flix/language/ast/Kind.scala

```diff
@@ -146,4 +146,19 @@ object Kind {
     case Arrow(k1, k2) => k1 :: kindArgs(k2)
     case _ => Nil
   }
+
+  def hasError(k: Kind): Boolean = k match {
```

Comment:
Add comment

Reply:
Thinking we should either have `containsError`, `isError` or `hasError`. What do you prefer?

Reply:
I think hasError is OK. Just need a comment